core/tests/protractor_desktop/coreEditorAndPlayerFeatures.js

```diff
@@ -33,6 +33,148 @@ var ExplorationEditorPage =
 var ExplorationPlayerPage =
   require('../protractor_utils/ExplorationPlayerPage.js');
 var LibraryPage = require('../protractor_utils/LibraryPage.js');
+var ExplorationEditorPage =
+  require('../protractor_utils/ExplorationEditorPage.js');
+
+describe('Enable correctness feedback and set correctness', function() {
+  var explorationEditorPage = null;
+  var explorationEditorMainTab = null;
+  var explorationEditorSettingsTab = null;
+  var explorationPlayerPage = null;
+  var currentExplorationIndex = 0;
+  var EXPLORATION_TITLE = 'Dummy Exploration';
+  var explorationTitle = null;
+  var libraryPage = null;
+  var correctOptions = [
+    ['MultipleChoiceInput', 'Correct!'],
+    ['TextInput', 'One'],
+    ['NumericInput', 3]
+  ];
+
+
+  var applyCommonSettings = function() {
+    explorationEditorPage.navigateToSettingsTab();
+    explorationEditorSettingsTab.setTitle(explorationTitle);
+    explorationEditorSettingsTab.setCategory('Algorithm');
+    explorationEditorSettingsTab.setObjective('Learn more about Oppia');
+    explorationEditorSettingsTab.setLanguage('English');
+    explorationEditorSettingsTab.enableCorrectnessFeedback();
+  };
+
+  beforeAll(function() {
+    users.createUser('user@markCorrect.com', 'userMarkCorrect');
+    users.login('user@markCorrect.com');
+  });
+
+  beforeEach(function() {
+    explorationTitle = EXPLORATION_TITLE + String(currentExplorationIndex);
+    libraryPage = new LibraryPage.LibraryPage();
+    explorationPlayerPage = new ExplorationPlayerPage.ExplorationPlayerPage();
+    explorationEditorPage = new ExplorationEditorPage.ExplorationEditorPage();
+    explorationEditorMainTab = explorationEditorPage.getMainTab();
+    explorationEditorSettingsTab = explorationEditorPage.getSettingsTab();
+    creatorDashboardPage = new CreatorDashboardPage.CreatorDashboardPage();
+    workflow.createExploration();
+  });
+
+  it('should allow selecting correct feedback from the response editor ' +
```

Comment:
Reading this test suite I think it is ok to keep the applyCommonSettings function separate. You may ignore the above comment to move it into beforeEach().